src/chttpd_db.erl

```diff
@@ -83,7 +83,8 @@ handle_changes_req(#httpd{path_parts=[_,<<"_changes">>]}=Req, _Db) ->
 handle_changes_req1(#httpd{}=Req, Db) ->
     #changes_args{filter=Raw, style=Style} = Args0 = parse_changes_query(Req),
     ChangesArgs = Args0#changes_args{
-        filter_fun = couch_changes:configure_filter(Raw, Style, Req, Db)
+        filter_fun = couch_changes:configure_filter(Raw, Style, Req, Db),
+        db_open_options = [{user_ctx, Db#db.user_ctx}]
```

Comment:
To be honest I pulled it from Db exactly for consistency reason. First, this is what we've been doing previously, i.e. passing Db and _its_ user context and propagating UserCtx. Second, we are opening db in fabric with db open options, so for me it seems more natural to do that with the same options that was used to open db here, rather than pull context from request record. Plus if (when) we move to Cowboy it'll be one less place to chase and change.